config/params/loader.go

```diff
@@ -59,19 +58,28 @@ func LoadChainConfigFile(chainConfigFileName string, conf *BeaconChainConfig) {
 	yamlFile = []byte(strings.Join(lines, "\n"))
 	if err := yaml.UnmarshalStrict(yamlFile, conf); err != nil {
 		if _, ok := err.(*yaml.TypeError); !ok {
-			log.WithError(err).Fatal("Failed to parse chain config yaml file.")
```

Comment:
breaking these methods up also allowed me to change them to return errors, so those can be handled by the caller instead of using Fatal logs.